From 908f7ecc8bba4602ba21a8c9376a53d6f2906b51 Mon Sep 17 00:00:00 2001
From: Sam Whited <sam@samwhited.com>
Date: Tue, 16 Jun 2020 12:27:37 -0400
Subject: [PATCH] Fix int/string conversion in error message

In Go 1.14 this would do the wrong thing silently, but this will cause
tests to fail on Go 1.15.
---
 pkg/fixtures/fixtures.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pkg/fixtures/fixtures.go b/pkg/fixtures/fixtures.go
index 83bc1072..3c04ad03 100644
--- a/pkg/fixtures/fixtures.go
+++ b/pkg/fixtures/fixtures.go
@@ -95,7 +95,7 @@ func NewFixture(fs afero.Fs, apiKey, stripeAccount, baseURL, file string) (*Fixt
 	}
 
 	if fxt.fixture.Meta.Version > SupportedVersions {
-		return nil, fmt.Errorf("Fixture version not supported: %s", string(fxt.fixture.Meta.Version))
+		return nil, fmt.Errorf("Fixture version not supported: %d", fxt.fixture.Meta.Version)
 	}
 
 	return &fxt, nil
